frontend/src/components/maps/leaflet/PointClusterer.tsx

```diff
@@ -361,7 +361,7 @@ export const PointClusterer: React.FC<PointClustererProps> = ({
           !currentClusterIds.includes(+selected?.parcelDetail?.id) && (
             <SelectedPropertyMarker
               {...selected.parcelDetail}
-              icon={getMarkerIcon({ properties: selected } as any)}
+              icon={getMarkerIcon({ properties: selected } as any, true)}
```

Comment:
I'm stumped for what to do for the spiderfied clusters

Reply:
Do you mean highlighting the selected cluster?  Is that a requirement?  I'd lean towards not doing that, as the value doesn't seem to justify it.

Reply:
No, I mean highlighting a pin within a cluster, as this only will highlight single pins

Reply:
Ah, that would be more complicated yes.

Reply:
I think just highlighting the **selected** pin is fine, I've made some updates that should make it appear more consistently.